lib/components/eureka_list_view.dart

```diff
@@ -55,7 +55,7 @@ class _EurekaListViewState extends State<EurekaListView> {
     // this is used to format data to return " X days/hours/minutes ago"
     final dateTime =
         DateTime.parse(widget.filteredQuestionsList[index].questionDate)
-            .subtract(new Duration(minutes: 1));
+            .subtract(new Duration(hours: -7));
```

Comment:
Is this value hardcoded to only work for Pacific Time Zone? If it is, I'm not sure if this is the best solution.

Reply:
Issue is bigger  than it seems, it will stay hardcoded at the moment. New task can be created to adjust it, if we have time